### ISSUE TITLE
Slides per view not working on Android (Galaxy S5 & Galaxy S8 at a minimum)

### ISSUE THREAD

**Issue by JakubSvihla** [nolimits4web/swiper#2723]
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Swiper Version: 4.3.3.
* Platform/Target and Browser Versions:  ANDROID (Galaxy S5 & Galaxy S8)

### What you did
breakpoints: {
    1024: {
      loop: true,
      slidesPerView: 1,
      simulateTouch: true
    }
  }

### Expected Behavior
I expected to see only 1 slide.

### Actual Behavior
Instead, 5 slides are displaying, which is the total number of slides.

Testing on actual device Galaxy S8, when you start scrolling, it resolves itself to 1 slide actually, but the navigation arrows are unclickable.

Testing on Galaxy S5 in Chrome devtools emulator, when scrolling I still see 5 slides.

For some reason, there is an inline style="width: 72px" (roughly 1/5 of the screen) on android and an inline style="width: 375px" (roughly full-width) on iOS on each individual slide.

When I unclick that inline 72px in the dev tools, things start to appear normal, but the nav buttons are still unclickable.

No problems on iOS.

**Comment by nolimits4web** [comment]
`loop` doesn't well supported with breakpoints

**Comment by JakubSvihla** [comment]
even if i disable the loop, it still happens :(

**Comment by nolimits4web** [comment]
There could be something else, do you have a live example of the issue?

**Comment by JakubSvihla** [comment]
This is the problem, I don't have one at the moment. I am going to make one and post a link over the weekend.

**Comment by JakubSvihla** [comment]
I seem to have found a "solution", more of a temporary fix: 

slidesPerView: (function() {
    if (mobileOS() === "android") {
      console.log("android");
      return 1;
    }
    return 5;
  })(),

**Comment by joejoseph00** [comment]
Swiper is a great library, amazing, looking forward to the next release. 

Meanwhile, I have released a forked version of swiper and I tagged the release v4.4.2
it resolves the slidesPerView issues in mobile.
https://github.com/entreprise7pro/swiper/releases/tag/v4.4.2

otherwise, hopefully we will see a new release of swiper very soon from @nolimitsweb , great job on this library, it's the best one I've seen.
